public/javascripts/data/user.js

```diff
@@ -32,15 +33,16 @@ exports.writeUserToDatabase = (username, password) => {
     return;
 };
 
-
+// Function compares user inputted login credentials to database
 exports.tryLogin = (username, password) => {
 
     const db = new Database('public/javascripts/data/stories.db');
+    
     // Check for valid user name
     const userStmt = db.prepare('SELECT user_name FROM users WHERE user_name = (?)');
     const userNameValue = userStmt.get(username);
+    
     if(!userNameValue) {
-        console.log(`That user name is not in the database ${username}`)
         db.close();
         return false
```

Comment:
Once you have a combined SQL statement, you can do something like:

```js
let user = userStmt.get(username, hashed_password)
db.close();
return user;
```

Again prefer `let` over `const`